packages/app/src/styles.ts

```diff
@@ -3,4 +3,4 @@
    Output is later concatenated with local app styles. */
 
 import 'react-reflex/styles.css';
-import '@h5web/lib/dist/styles.css'; // local + global lib styles
+import '@h5web/lib/dist/styles.css'; // distributed lib styles
```

Comment:
While the import path doesn't change, it is now properly defined in the lib's `package.json` via  `"exports"`.